libs/spark/src/Unstable_Button/Unstable_Button.stories.tsx

```diff
@@ -1,45 +1,41 @@
 import * as React from 'react';
 import type { Meta, Story } from '@storybook/react/types-6-0';
 import { Unstable_Button, Unstable_ButtonProps } from '..';
-import { Unstable_ChevronDown, Unstable_AlertOctagon } from '../internal';
-import { containFocusIndicator, sparkThemeProvider } from '../../stories';
+import {
+  ChevronDown,
+  containFocusIndicator,
+  Plus,
+  sparkThemeProvider,
+} from '../../stories';
 
-// eslint-disable-next-line @typescript-eslint/no-empty-interface
-interface Sb_Unstable_ButtonProps
-  extends Omit<Unstable_ButtonProps, 'tabIndex'> {}
-
-export const Sb_Unstable_Button = (props: Sb_Unstable_ButtonProps) => (
-  <Unstable_Button {...props} />
-);
+export const _retyped = Unstable_Button as typeof Unstable_Button;
```

Comment:
Why do this?

Reply:
the auto-controls storybook plugin has trouble picking up prop types of default exports. so sometimes you have to re-export as a named export and use that instead, but also re-apply the props/typing. this is the shortest way ive found to do so.